release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1145,6 +1145,51 @@ With this update, the Ingress Operator no longer adds the `clientca-configmap` f
 [id="ocp-4-15-openshift-cli-bug-fixes"]
 ==== OpenShift CLI (oc)
 
+[discrete]
+[id="ocp-4-15-openshift-oc-mirror-bug-fixes"]
+==== oc-mirror
+
+Prior to this release, to filter operator packages by channel, for example, `mirror.operators.catalog.packages.channels`, you had to specify the default channel for the package, even if you did not intend to use the packages from that channel. Based on this information, the resulting catalog is considered invalid if the `imageSetConfig` does not contain the default channel for the package.
+
+[WARNING]
+====
+Modifying the default channel should not be taken lightly as it could have an impact on the Operator's upgrade paths. The use of `defaultChannel` is the responsibility of the end user.
```

Comment:
`The use of defaultChannel is the responsibility of the end user.` This can be replaced by when the user must be using.

```suggestion
Modifying the default channel can have an impact on the Operator's upgrade paths. The use of `defaultChannel` is the responsibility of the end user.
```

Reply:
ack.